prime-router/docs/observability/local-dev-setup.md

```diff
@@ -33,6 +34,13 @@
     4. Wait about 10 minutes and ensure data is flowing into the appinsights portal. You should be able to see basic
        statistics around the requests.
 
+## Troubleshooting
```

Comment:
On my mac I ran into an issue with opening the jar, might be nice to add a note:

If you encounter an issue with java unable to open the JAR:
- Double-check your path in `settings.local.json` the path should be in escaped quotes \"/path/to/file\"
- Double-click the jar in finder, then go to System Settings -> Security and click "Open Anyway".
  - Alternative is to use the cli tool to remove the quarantine attr, something like this ```xattr -dr com.apple.quarantine <path to file>/my-binary-amd64```
- Redownload the .jar file (you can attempt extracting it with the `unzip` cli command)

Reply:
Added